projects/frontend/data-pipelines/gui/cypress.json

```diff
@@ -23,7 +23,6 @@
   "env": {
     "hars_folders": "e2e/hars",
     "csp_url": "https://console-stg.cloud.vmware.com",
-    "login_url": "https://console-stg.cloud.vmware.com/csp/gateway/am/api/auth/api-tokens/authorize",
```

Comment:
I don't understand why splitting this config to two ($csp_url + path) - because the path itself is not universal, it is rather CSP-specific, as well as the rest of the URL -> `cspschema://csp-domain-name/csp/path/`.

The result is -now we have two CSP-specific configs in favour of one in one place. Also, we still have `csp_url` env property name, that is also CSP-specific.

Reply:
I splitted in two because same domain is used in two places in commands.js file, one to request for JWT Token and the other to request claims expansion.
Instead having two different fields here that refer to same domain and same service with different APIs, I added domain and pathnames to APIs are hardcoded in the script file commands.js

Reply:
The approach explained makes a priorly configurable value - partially hardcoded now.
The point is, those pathnames shouldn't be hardcoded, because they are a part of the OAuth provider config.